index.html

```diff
@@ -0,0 +1,475 @@
+                    eiusmod
+                    tempor incididunt ut labore et
+                    dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi
+                    ut
+                    aliquip
+                    ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse
+                    cillum
+                    dolore eu
+                    fugiat nulla pariatur. ex ea commodo consequat.</p>
+
+
+
+
+
+        </article>
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+        <article class="article3">
+
+
+
+            <h1>Skills</h1>
+
+
+
+            <article class="article3_right"></article>
+
+
+
+
+            <article class="article3_left">
+                <div class="article3_left_backendlogo"><img src="./img/icon_backend.jpg" alt="image"></div>
+                <h1 class="article3_left_backend">Back End</h1>
+                <p class="article3_left_php">PHP, Node.js</p>
+
+
+                <div class="article3_right_line1"></div>
+
+                <div class="article3_left_frontendlogo"><img src="./img/icon_frontend.jpg" alt="image"></div>
+                <h1 class="article3_left_frontend">Front End</h1>
+                <p class="article3_left_html">Html, CSS, JavaScript</p>
+
+
+                <div class="article3_right_line2"></div>
+
+                <div class="article3_left_uxuilogo"><img src="./img/icon_UxUi.jpg" alt="image"></div>
+                <h1 class="article3_left_uxui">UX/UI</h1>
+                <p class="article3_left_figma">Figma</p>
+            </article>
+
+
+
+        </article>
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+        <article class="article4">
+            <h1>Portfolio</h1>
+
+
+
+            <img>
+            <img>
+            <img>
+            <img>
+            <img>
+
+
+
+
+            <h1 class="article4_Design">Design</h1>
+
+            <p class="article4_Lorem"> Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod
+                tempor incididunt ut labore et
+                dolore magna aliqua.
+            </p>
+
+
+
+
+            <button class="article4_view1" href="">
+
+
+
+
+                <a href="">View</a>
+
+
+
+
+            </button>
+
+
+
+
+
+
+            <h1 class="article4_Designproject">Design project</h1>
+
+            <p class="article4_xcepteur"> Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia
+                deserunt mollit anim id est laborum
+            </p>
+
+
+
+
+
+            <button class="article4_view2">
+
+                <a href=""> View</a>
+            </button>
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+        </article>
+
+
+
+
+
+
+
+
+
+
+
+
+        <article class="article5">
+
+            <h1>Contact us</h1>
+
+            <div class="article5_main">
+
+
+                <div class="article5_main_left">
+
+
+
+                    <img class="article5_main_right_envelope" src="./img/envelope.png" alt="image">
```

Comment:
Kolejny raz widzę `alt="image"`. Atrybut `alt` jest po to aby opisać w nim co dokładnie prezentuje zdjęcie, jest to duże ułatwienie dla osób które korzystają z czytników ekranowych (osoby niedowidzące). Samo `image` nic im nie mówi.